src/book/mod.rs

```diff
@@ -262,6 +262,10 @@ impl MDBook {
         let mut index = File::create(&themedir.join("index.hbs"))?;
         index.write_all(theme::INDEX)?;
 
+        // toc.hbs
+        let mut toc = File::create(&themedir.join("toc.hbs"))?;
```

Comment:
From memory `File::create()` takes anything which is `AsRef<Path>`, so you should be able to drop the `&`'s from `File::create(&themedir.join("toc.hbs"))` and friends.